src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -244,15 +246,42 @@ def listDatasetFiles(self, datasetPath):
         Get list of files for dataset
 
         """
-        return [ x['logical_file_name'] for x in self.dbs.listFileArray(dataset = datasetPath)]
+        return [x['logical_file_name'] for x in self.dbs.listFileArray(dataset = datasetPath)]
 
-    def listDatatiers(self):
+    @staticmethod
+    def listDatatiers(dbsUrl=None):
```

Comment:
since there dbsUrl is mandatory value it shouldn't have default value set. Anyway, this is minor. I will merge the patch